src/main/java/com/duckblade/osrs/toa/features/ChestAudio.java

```diff
@@ -0,0 +1,181 @@
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.duckblade.osrs.toa.features;
+
+import com.duckblade.osrs.toa.TombsOfAmascutConfig;
+import com.duckblade.osrs.toa.module.PluginLifecycleComponent;
+import java.io.File;
+import java.io.IOException;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import javax.sound.sampled.AudioInputStream;
+import javax.sound.sampled.AudioSystem;
+import javax.sound.sampled.Clip;
+import javax.sound.sampled.FloatControl;
+import javax.sound.sampled.LineUnavailableException;
+import javax.sound.sampled.UnsupportedAudioFileException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import net.runelite.api.DynamicObject;
+import net.runelite.api.GameObject;
+import net.runelite.api.events.GameObjectDespawned;
+import net.runelite.api.events.GameObjectSpawned;
+import net.runelite.api.events.GameTick;
+import net.runelite.client.RuneLite;
+import net.runelite.client.eventbus.Subscribe;
+import net.runelite.client.events.ConfigChanged;
+
+@Slf4j
+@Singleton
+@RequiredArgsConstructor(onConstructor_ = @Inject)
+public class ChestAudio implements PluginLifecycleComponent
+{
+	private static final int SARCOPHAGUS_OBJECT_ID = 46220;
+	private static final int SARCOPHAGUS_OBJECT_ID_2 = 46221;
+	private static final int OPENING_ANIMATION = 9505;
+
+	private Clip clip = null;
+	private GameObject chest =  null;
+
+	@Inject
+	private TombsOfAmascutConfig config;
+
+	@Override
+	public boolean isConfigEnabled(TombsOfAmascutConfig config)
+	{
+		return config.chestAudioEnable();
+	}
+
+	@Override
+	public void startUp()
+	{
+		// Nothing to do on startUp but its required for `PluginLifecycleComponent`
+	}
+
+	@Override
+	public void shutDown()
+	{
+		if (clip != null)
+		{
+			clip.close();
+		}
+		clip = null;
+	}
+
+	@Subscribe
+	public void onConfigChanged(ConfigChanged c)
+	{
+		if (!c.getGroup().equals(TombsOfAmascutConfig.CONFIG_GROUP))
+		{
+			return;
+		}
+
+		if (c.getKey().equals("chestAudioVolume"))
```

Comment:
replace with string constant in config class defined right before the config item (and use in the config item annotation)